probe/docker/container.go

```diff
@@ -277,12 +277,15 @@ func (c *container) ports(localAddrs []net.IP) report.StringSet {
 			continue
 		}
 		for _, b := range bindings {
-			if b.HostIP == "0.0.0.0" {
-				for _, ip := range localAddrs {
+			if b.HostIP != "0.0.0.0" {
```

Comment:
An address different than `"0.0.0.0"` indicates IPv6? I think a comment is in order.